Platon/backend/app/follow_system/forms.py

```diff
@@ -6,31 +6,43 @@
 # following: user that is followed by someone
 class GetFollowingsForm(Form):
     follower_id = IntegerField('follower_id', validators=[validators.DataRequired()])
+    page = IntegerField("page")
+    per_page = IntegerField("per_page")
 
 get_followings_parser = reqparse.RequestParser()
 get_followings_parser.add_argument('follower_id', required=True, type=int,
                                    help="ID of the follower. Follower is the one who follows someone.",
                                    location='form')
+get_followings_parser.add_argument('page',type=int,help="Page index that you want(Starts from 0)",location='form')
+get_followings_parser.add_argument('per_page',type=int,help="Number of items in a page",location='form')
```

Comment:
Why did you took this parameters form the body? As I know there is no body content in GET/HEAD requests.